<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FinalProjectWithGUIClass</class>
 <widget class="QWidget" name="FinalProjectWithGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FinalProjectWithGUI</string>
  </property>
  <widget class="QPushButton" name="downloadButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Download data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="printButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Print the data for one stock</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showGraphButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Show CAARs graph</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calculateButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate AARs and CAARs</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit the Program</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showCAARButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Show AARs or CAARs for one group</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="outputBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>371</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="welcomeLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>711</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Welcome to the automatic stock analysis program published by one of the best groups!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="inputLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Please input here:</string>
   </property>
  </widget>
  <widget class="QLabel" name="outputLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Output:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputLine">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="group1Button">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Group 1, 'miss'</string>
   </property>
  </widget>
  <widget class="QPushButton" name="group2Button">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>90</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Group 2, 'match'</string>
   </property>
  </widget>
  <widget class="QPushButton" name="group3Button">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>90</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Group 3, 'beat'</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submitStockPrice">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>90</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AARButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>90</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>AAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CAARButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>90</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>CAAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="group1AAR">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Group 1, 'miss'</string>
   </property>
  </widget>
  <widget class="QPushButton" name="group2AAR">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>90</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Group 2, 'match'</string>
   </property>
  </widget>
  <widget class="QPushButton" name="group3AAR">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>90</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Group 3, 'beat'</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="FinalProjectWithGUI.qrc"/>
 </resources>
 <connections/>
</ui>
